### DISCUSSION TITLE
Paragraphs that are fragments with in-line markdown

### DISCUSSION THREAD

**Discussion by matthewturk** [ebullient/obsidian-slides-extended#65] [Q&A] [answered]
Hi!  I've got a series of paragraphs that I'd like to show up as fragments, but I also sometimes use markdown in them.  So for instance:

```
First thing <!-- .element: class="fragment" -->

Second *thing* is this <!-- .element: class="fragment" -->
```

starts with the text "Second          is this" and then the first paragraph and then the italicized item are added as fragments.  I get around this by using `<p class="fragment">` wrappers, but I'm wondering if inside slides-extended there is a better approach.

(For instance, I only just learned about using `+` in a list to make it fragmented, and that *rules*.)

Is this the best way to do it, or am I missing something?  Would a `list-decoration: none;` CSS style and then using fragmented lists be better, or is that opening up a bunch of weird formatting issues?

**Comment by ebullient** [comment, marked as answer]
In these cases, I either surround the text with a `<span>...</span>`, or I use a block: 

```
::: block
First thing
::: <!-- .element: class="fragment" -->

::: block
Second *thing* is this
::: <!-- .element: class="fragment" -->
```

**Comment by matthewturk** [comment]
This makes perfect sense -- thank you.
